packages/host/app/services/card-service.ts

```diff
@@ -150,4 +150,18 @@ export default class CardService extends Service {
     }
     return results;
   }
+
+  async getFields(
+    card: Card
+  ): Promise<{ [fieldName: string]: Field<CardConstructor> }> {
+    await this.apiModule.loaded;
+    return this.api.getFields(card, { includeComputeds: true });
+  }
+
+  async isPrimitive(
+    card: Card
```

Comment:
the way you are using this is that you are passing in a card definition (`typeof Card`). You can certainly have a method override so that this function works for both card instances  (`Card`) and card definitions (`typeof Card`). but the way you have it now is problematic